Python/Tests/Core.UI/FormattingUITests.cs

```diff
@@ -66,7 +66,8 @@ public class FormattingUITests {
     pass
 
 
-def g(): return 5 ** 2
+def g():
+    return 5 ** 2
```

Comment:
i was going to add another fix for black formatting but instead just updating the test case expected result